app/models/group.rb

```diff
@@ -0,0 +1,8 @@
+class Group < ActiveRecord::Base
+	has_many :comments
+	validates_presence_of :book
```

Comment:
I was getting my wires crossed and I thought this was referring to a `Book` instance from the other PR. Maybe it should? I'm not sure if that's what we want for this though...